lancache-litmus/overlay/etc/nginx/sites-available/litmus.conf.d/10_litmus.conf

```diff
@@ -0,0 +1,3 @@
+server_name lancache LITMUS_HOSTNAME;
```

Comment:
can we set this to default

Reply:
LITMUS_HOSTNAME defaults to lancache.lan from the Dockerfile